core/src/components/searchbar/searchbar.ionic.scss

```diff
@@ -49,25 +60,98 @@
 // -----------------------------------------
 
 .searchbar-clear-button {
-  @include globals.position(50%, globals.$ionic-space-200, null, null);
+  @include globals.position(
+    searchbarVars.$searchbar-ionic-input-clear-icon-size,
+    searchbarVars.$searchbar-ionic-input-clear-icon-size,
+    null,
+    null
+  );
 
-  position: absolute;
+  width: searchbarVars.$searchbar-ionic-input-clear-icon-size;
+  height: searchbarVars.$searchbar-ionic-input-clear-icon-size;
 
-  width: globals.$ionic-scale-600;
-  height: globals.$ionic-scale-600;
+  background-color: transparent;
 
-  transform: translateY(-50%);
+  font-size: globals.$ionic-font-size-400;
```

Comment:
Should this use scale?

Reply:
https://github.com/ionic-team/ionic-framework/pull/29917/commits/8eb73311e59b8d57c7e169e4540fd873ae812f26